### ISSUE TITLE
do you support multi-labels detection?

### ISSUE THREAD

**Issue by KungFuPandaPro** [ultralytics/ultralytics#15913]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

if one person have many labels,for  example person、man

**Comment by glenn-jocher** [comment]
@KungFuPandaPro yes, YOLOv8 supports multi-label detection. You can train your model on a dataset where each object can have multiple labels, and the model will predict all applicable labels for each detected object.

**Comment by KungFuPandaPro** [comment]
> @KungFuPandaPro yes, YOLOv8 supports multi-label detection. You can train your model on a dataset where each object can have multiple labels, and the model will predict all applicable labels for each detected object.

so happly to get your reply,but how can i use that

**Comment by KungFuPandaPro** [comment]
> @KungFuPandaPro yes, YOLOv8 supports multi-label detection. You can train your model on a dataset where each object can have multiple labels, and the model will predict all applicable labels for each detected object.
so happly to get your reply,but how to use that
@glenn-jocher

**Comment by glenn-jocher** [comment]
To use multi-label detection with YOLOv8, ensure your dataset annotations include all applicable labels for each object. Then, train your model as usual. For detailed guidance, refer to the [Ultralytics documentation](https://docs.ultralytics.com/datasets/detect/).

**Comment by KungFuPandaPro** [comment]
> To use multi-label detection with YOLOv8, ensure your dataset annotations include all applicable labels for each object. Then, train your model as usual. For detailed guidance, refer to the [Ultralytics documentation](https://docs.ultralytics.com/datasets/detect/).

does yolov8 support?

**Comment by KungFuPandaPro** [comment]
@glenn-jocher  does yolov8 support?

**Comment by KungFuPandaPro** [comment]
@glenn-jocher  I use the annotation format 1 0.529 0.4849397590361446 0.262 0.9397590361445783 0,but it does not work,model only learn one label

**Comment by KungFuPandaPro** [comment]
![1725246806611](https://github.com/user-attachments/assets/0b24d309-01f3-49a9-9c84-e18ffd6710f8)
I use the infer code,but the probs always None  @glenn-jocher

**Comment by KungFuPandaPro** [comment]
does YOLOv8  support multi-label classification within a single bounding box? @glenn-jocher

**Comment by glenn-jocher** [comment]
@KungFuPandaPro yOLOv8 does not natively support multi-label classification within a single bounding box. Each bounding box is associated with a single class label. For multi-label classification, you may need to implement a custom solution or consider alternative models that support this feature.

**Comment by KungFuPandaPro** [comment]
> @KungFuPandaPro yOLOv8 does not natively support multi-label classification within a single bounding box. Each bounding box is associated with a single class label. For multi-label classification, you may need to implement a custom solution or consider alternative models that support this feature.

thank you for your reply,I plan to implement a custom solution,but what the inference  code now can get the probs of each class

**Comment by glenn-jocher** [comment]
@KungFuPandaPro yOLOv8's inference code does not provide probabilities for each class within a single bounding box. You might need to modify the model's architecture and post-processing steps to achieve this.

**Comment by hu874** [comment]
> > @KungFuPandaProyOLOv8 本身不支持单个边界框内的多标签分类。每个边界框都与单个类别标签相关联。对于多标签分类，您可能需要实现自定义解决方案或考虑支持此功能的替代模型。
> 
> 谢谢你的回复，我计划实现一个自定义解决方案，但是现在的推理代码可以得到每个类的概率

Have you achieved it? I also want to achieve it
